torch/_torch_docs.py

```diff
@@ -1620,18 +1620,19 @@ def merge_dicts(*dicts):
            r"""
 .. function:: div(input, other, out=None) -> Tensor
 
-Divides each element of the input :attr:`input` with the scalar :attr:`other`
-and returns a new resulting tensor.
+Divides each element of the input ``input`` with the scalar ``other`` and
+returns a new resulting tensor.
 
 .. math::
     \text{out}_i = \frac{\text{input}_i}{\text{other}}
 
-If :attr:`input` is of type `FloatTensor` or `DoubleTensor`, :attr:`other`
-should be a real number, otherwise it should be an integer
+If ``input`` is of dtype :attr:`torch.float` or :attr:`torch.double`, ``other``
+should be a real number, otherwise it should be an integer. If ``input`` is
+integral and ``other`` is zero, the behavior is undefined and arbitrary.
```

Comment:
Right, the undefined (segfaulting) behavior is only when both input and other are integral and other is `0`. `floattensor/0` or `inttensor/0.0` both result in float tensors of `inf`

Reply:
#26538

Reply:
@nairbv Let me know if I need to change anything. Thanks!

Reply:
@xudev, for the general explanation maybe something closer to what we have on the release branch (https://github.com/pytorch/pytorch/pull/27501/) since "otherwise it should be an integer" is inaccurate.

The last sentence is incorrect because `torch.tensor(1)/0.0` is defined.

Reply:
@nairbv Thanks, I've updated the PR. I corrected/improved some markups and incorporated the two versions of text together, with your correction also handled.